ironfish-cli/src/commands/chain/sync.ts

```diff
@@ -7,7 +7,7 @@ import { IronfishApi } from '../../api'
 import { IronfishCommand } from '../../command'
 import { RemoteFlags } from '../../flags'
 
-const MAX_UPLOAD = 1000
+const MAX_UPLOAD = !isNaN(Number(process.env.MAX_UPLOAD)) ? Number(process.env.MAX_UPLOAD) : 100
```

Comment:
I'd pitch this form, as its less work and (IMO) more explicit.

```
const RAW_MAX_UPLOAD = Number(process.env.MAX_UPLOAD)
const MAX_UPLOAD = isNaN(RAW_MAX_UPLOAD) ? 100 : RAW_MAX_UPLOAD
```

Reply:
Noted, and included. Thanks!